tests/index.js

```diff
@@ -0,0 +1,42 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const app = require('../index')
+
+chai.use(chaiHttp);
+chai.should();
+describe("Highscores", () => {
+  describe("GET/", () => {
+
+    it("should get highscore data for jZERKk", (done) => {
+      chai.request(app)
+        .get('/jZERKk')
+        .end((err, res) => {
+          res.should.have.status(200)
+          res.body.should.be.a('object')
+          done()
+        })
+    })
+
+    it("should fail when not provided a username", (done) => {
+      chai.request(app)
+        .get('/')
+        .end((err, res) => {
+          res.should.have.status(404)
+          done()
+        })
+    })
```

Comment:
This will most likely need to be changed in the future.

Once we support no entering a username, this test will need to be updated

Reply:
I didn't see this right away, would have saved me some troubles. Whoops.